### DISCUSSION TITLE
Is it possible to not show notification automaticaly when app is on background state ?

### DISCUSSION THREAD

**Discussion by Ddasb** [invertase/react-native-firebase#6246] [Q&A] [answered]
I'm using notifee to handle notification. But rn firebase cloud message automaticaly handle message with notification key when i'm sending from google cloud messaging service. Is it possible to no display the automatic notification and let notife handle it ?

**Comment by mikehardy** [comment]
I think you want this? https://notifee.app/react-native/docs/ios/remote-notification-support

**Comment by Ddasb** [comment, marked as answer]
This only works for ios. But i used Firebase api instead of firebase console and it's works.
